src/views/RuleEngine/components/ResourceDialog.vue

```diff
@@ -114,13 +116,14 @@
 <script>
 import EmqSelect from '~/components/EmqSelect'
 import ArrayEditor from '~/components/ArrayEditor'
+import FileEditor from "../../../components/FileEditor"
```

Comment:
A unified import path is needed here, Please modify it to an absolute path, we will refactor and migrate in the follow-up, thanks! :)

```js
import FileEditor from "~/components/FileEditor"
```

Reply:
:+1: